src/QsCompiler/BondSchemas/SandboxInput/SandboxInput.bond

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "EntryPoint/EntryPoint.bond";
+namespace Microsoft.Quantum.QsCompiler.BondSchemas.SandboxInput;
+
+struct Bytecode
+{
+    5: blob Data;
+}
+
+struct Input
```

Comment:
A more descriptive name for this `struct` would be `QirExecutionWrapper`. I would change the name of the folder containing the *.bond file to _Qir_ and the file to _QirExecutionWrapper.bond_.